src/state/registers.ts

```diff
@@ -361,6 +362,10 @@ class ClipboardRegister extends Register implements Register.Readable,
   public readonly flags = Register.Flags.CanRead | Register.Flags.CanWrite;
 
   public async get() {
+    if (!vscode.workspace.getConfiguration(extensionName).get<boolean>("useSystemClipboard")) {
```

Comment:
Given how often the clipboard might be accessed, I would prefer subscribing to changes to that config value in the `Registers` constructor, similarly to what `Modes` does: https://github.com/71/dance/blob/64bf13e5fab09961952857beae3a2c09f600bff2/src/state/modes.ts#L575-L582

And storing a boolean in `ClipboardRegister`, like `_useSystemClipboard`.